keyboards/keebio/iris/keymaps/shinze/rules.mk

```diff
@@ -0,0 +1,6 @@
+RGBLIGHT_ENABLE = yes
+BACKLIGHT_ENABLE = yes
+
+ifndef QUANTUM_DIR
```

Comment:
Please remove this block - it's a holdover from the old Make system and no longer does anything.